go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/minio/dperf v0.4.2
 	github.com/minio/highwayhash v1.0.2
 	github.com/minio/kes v0.20.0
-	github.com/minio/madmin-go v1.4.13
+	github.com/minio/madmin-go v1.4.15
```

Comment:
are you planning to change the code that is present inside MinIO and use the one that was borrowed into madmin-go?

Reply:
@harshavardhana I can send a separate PR to remove the `internal/kernel/kernel.go` from `minio` and start using the same function(s) from `madmin-go`. (Can do so in this PR also if you prefer that)

Reply:
Let's do it in this PR @anjalshireesh